### ISSUE TITLE
[Lunar - The Silver Star (J)] Game stucks...

### ISSUE THREAD

**Issue by BenjaminSiskoo** [Yabause/yabause#185]
Left Picture : Yabause 03.13.2016 (Software)
Right Picture : SSF 0.12 Beta R4

After the end of the speech of the little boy, the game stuck. With SSF, another character comes from the bottom of the screen.

![lunar the silver star](https://cloud.githubusercontent.com/assets/12481242/13729165/2e00737c-e92f-11e5-976d-7cfa588c9c04.JPG)

**Comment by cyberwarriorx** [comment]
This bug also happens with - Lunar Silver Star Complete MPEG (J)

**Comment by BenjaminSiskoo** [comment]
Yabause 0.9.15 : Same problem.

**Comment by mistydemeo** [comment]
Looks like this was fixed in uoyabause: https://github.com/devmiyax/yabause/commit/e5deeae8f02f33868a25c9d63972e5e171305e2e The patch doesn't apply cleanly because of rewrites to the SCSP code though.

**Comment by cyberwarriorx** [comment]
My concern about this patch is that it "partly improved" the issue according to the patch. Changes to the code in question could easily break other games if great care wasn't taken.

**Comment by mistydemeo** [comment]
Yeah, I haven't tested any other games. I can't build uoyabause because it doesn't build on OS X, but I briefly test-drove the first half of the patch (unconditionally setting the various `scsp_main_interrupt`s), and it fixed the freeze but did introduce other audio glitches.
